src/Uno.Extensions.Reactive/Collections/Tracking/CollectionAnalyzer.cs

```diff
@@ -175,15 +162,15 @@ public CollectionUpdater GetUpdater(RichNotifyCollectionChangedEventArgs arg, IC
 		*/
 
 		int added = 0, moved = 0, removed = 0;
-		var buffer = new ChangesBuffer(oldItems.Count, newItems.Count, eventArgsOffset);
+		var buffer = new ChangesBuffer(old.items.Count, @new.items.Count, eventArgsOffset);
 
-		var oldEnumerator = new SourceEnumerator<TCollection>(oldItems, indexOf);
+		var oldEnumerator = new SourceEnumerator(old.items, old.indexOf);
 		while (oldEnumerator.MoveNext())
 		{
 			var oldIndex = oldEnumerator.CurrentIndex;
 			var oldItem = oldEnumerator.Current!;
 			var resultIndex = oldIndex - removed + added + moved - oldEnumerator.Ignored; // The current index in the (virtual) result collection (i.e. oldIndex ignoring the removed/added items)
-			var newIndex = indexOf(oldItem, newItems, resultIndex);
+			var newIndex = @new.indexOf(oldItem, resultIndex, @new.items.Count - resultIndex);
```

Comment:
`@new` what ? Seems like a unnecessary reduction of name :)

Reply:
It was to avoid the redundant "items" in `newItems.items` ... but like for the previous comment, I've reworked this in my current branch and I've already restored the `newItems`:

![image](https://user-images.githubusercontent.com/8635919/173149215-651e1d53-2b3f-4a12-a650-d7df9093be4d.png)